<launch>
    <!-- Point Cloud to LiDAR Data Saver Node -->
    <node name="lidar_data_saver" pkg="depth_to_lidar" type="convert.py" output="screen">

        <param name="input_bag_filename" value="/path/to/your_bag/{input_bag_file_name}.bag" />

        <param name="input_depth_topic" value="{input/depth/topic/name}" />
        <param name="output_bag_filename" value="/path/to/save/{output_bag_file_name}.bag" />

        <param name="number_of_lidar_channels" value="32" />
        <param name="vertical_fov_lowest" value="-60" />
        <param name="vertical_fov_highest" value="60" />

    </node>
</launch>
